src/app/login/success/page.tsx

```
@@ -3,20 +3,35 @@
+      try {
+        // FCM 토큰 등록 (백그라운드)
+        registerFCMToken().catch((error) => {
+          console.warn('FCM registration failed, but continuing:', error);
+        });
+
+        const response = await getRoleAPI.getRole();
 
+        switch (response.content.role) {
+          case 'ROLE_NO_INFO':
+            router.push('/signup/privacy');
+          case 'ROLE_REPORTED':
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)